components/src/Guidelines/index.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { ReactNode } from 'react'
+import { Styled, Text as ThemeUIText } from 'theme-ui'
+import { Box } from 'theme-ui'
+
+export const Guidelines = ({ children }: ListProp) => (
+    <Box>
+        {children}
+    </Box>
+)
+
+const Dos = ({ children, title, icon }: ListProps) => (
+    <ThemeUIText
+        sx={{
+            fontFamily: 'sans serif',
+            lineHeight: 2,
+            letterSpacing: 1,
+
+        }}>
+
+        <ThemeUIText as="h3">{icon}{title}</ThemeUIText>
+        <Styled.ul>
+            {children}
+        </Styled.ul>
+
+    </ThemeUIText>
+)
+
+const Donts = ({ children, title, icon }: ListProps) => (
+    <ThemeUIText
+        sx={{
+            fontFamily: 'sans serif',
+            lineHeight: 2,
+            letterSpacing: 1,
+        }}>
+
+        <ThemeUIText as="h3">{icon}{title}</ThemeUIText>
+        <Styled.ul>
+            {children}
+        </Styled.ul>
+
+    </ThemeUIText>
+)
+
+interface ListProps {
+    children: ReactNode
+    icon: ReactNode
+    title: string
```

Comment:
Refer back to #8; these attributes are not required, right? So you can add the `?` as shown to let the user know they're optional! :man_dancing:

Reply:
I forgot to show an example :sweat_smile:: `icon?: ReactNode`